auslib/db.py

```diff
@@ -345,13 +381,17 @@ def _prepareInsert(self, trans, changed_by, **columns):
         if self.onInsert:
             self.onInsert(self, "INSERT", changed_by, query, trans)
 
+        potential_required_signoffs = self.getPotentialRequiredSignoffs([columns], transaction=trans)
+        if potential_required_signoffs:
+            verify_signoffs(potential_required_signoffs, signoffs)
```

Comment:
I didn't end up doing this because of the signoffs refactor, but I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1332412.